Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -477,6 +477,125 @@ void TextureCacheBase::BindTextures()
   }
 }
 
+class ArbitraryMipmapDetector
+{
+private:
+  using PixelRGBA = std::array<u8, 4>;
+  using PixelRGBAf = std::array<float, 4>;
+
+  static float SRGBToLinear(u8 sRGBbyte) {
```

Comment:
I also kind of pointed this out several times now, but our coding style puts the brace on the next line. I really hate to say this, but this is kind of bordering on rude at this point; I don't feel like I should need to keep pointing this out.

Reply:
I'm sorry, I'm just really used to it and set up my tools to do that... given how many iterations this is going through, I also feel bad about you spending time reviewing intermediate code that might be throwaway. 
I'm really not trying to challenge the style btw! I'm absolutely fine with it. 
Your linter seems to be able to spot these things, do you know how I can run it offline?